README.md

```diff
@@ -69,6 +69,21 @@ try {
 RevAiJob revAiJob = apiClient.submitJobLocalFile(fileInputStream, String fileName, RevAiJobOptions options);
 ```
 
+You can also submit a job to be handled by a human transcriber using our [Human Transcription](https://www.rev.ai/docs#section/Human-Transcription-(Labs)) option.
+```
+String urlLinkToFile = "https://www.rev.ai/FTC_Sample_1.mp3";
+RevAiJobOptions options = new RevAiJobOptions();
+
+// set to perform human transcription
+options.setTranscriber("human");
+
+// optional job options
```

Comment:
Should segments to transcribe also be included here? The segment from 0 to 2 minutes should work with that file

Reply:
Potentially test mode as well

Reply:
Done